west.yml

```diff
@@ -142,7 +142,7 @@ manifest:
       groups:
         - hal
     - name: hal_ambiq
-      revision: ff4ca358d730536addf336c40c3faa4ebf1df00a
+      revision: pull/11/head
```

Comment:
The module PR is merged. Please replace with 705f1cbb1ccbdaac7613eb7b27c208c8f592e0c4

Reply:
@carlescufi done